modules/ossm-cr-example.adoc

```diff
@@ -187,9 +187,9 @@ spec:
       some-other-egress-gateway: {}
 
   policy:
-    type: Istiod # or Mixer
+    type: Mixer
     mixer: # only applies if policy.type: Mixer
-      enableChecks: false
+      enableChecks: true
```

Comment:
@dfennessy Is this a change that is specific for 3scale? Or is this a change you meant to make in this example for the entire product?

Reply:
+1 This is a service mesh example file, and I don't think we should be recommending using a deprecated component unless there is a need (that is, for 3scale).

Reply:
See point 3. I may have misread it: https://issues.redhat.com/browse/THREESCALE-6894?focusedCommentId=16253154&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16253154

Reply:
@neal-timpe  @JStickler 

Is the consensus is to change this to `false`?

Reply:
Our comments are less about if the setting is correct and more that this setting only applies to 3scale and should be documented in the 3scale section of the docs and not in the general custom resource example.

Reply:
Ah, I see. It came up as part of the original review with the SME and has since been signed off by QE. My initial thought would be not to change anything for now.

@unleashed and/or @phala might have some comment on it, however they might not be available at the moment.